packages/offix-scheduler/src/store/OfflineStore.ts

```diff
@@ -1,7 +1,8 @@
 import { PersistentStore, PersistedData } from "./PersistentStore";
 import { QueueEntryOperation, QueueEntry } from "../queue";
 import { OfflineStoreSerializer, DefaultOfflineSerializer } from "./OfflineStoreSerializer";
-
+import Debug from "debug";
+const debug = Debug("Store: ");
```

Comment:
I would go with the wrapper DataSync:Store: etc. Space after makes this unusable.

Reply:
We could go with `Datasync:Store`, but that will be the flag we have to use to make this information visible. i.e. we would have to run `DEBUG=DataSync:Store yarn start` or whatever script you are using.

Maybe we could shorten it and keep it descriptive still.

Something like:
`const info = debug('offix:store')` or const info = debug('sync:store')`